packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverBody/useTeachingPopoverBodyStyles.styles.ts

```diff
@@ -1,33 +1,76 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SlotClassNames } from '@fluentui/react-utilities';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { TeachingPopoverBodySlots, TeachingPopoverBodyState } from './TeachingPopoverBody.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const teachingPopoverBodyClassNames: SlotClassNames<TeachingPopoverBodySlots> = {
   root: 'fui-TeachingPopoverBody',
-  // TODO: add class names for all slots on TeachingPopoverBodySlots.
-  // Should be of the form `<slotName>: 'fui-TeachingPopoverBody__<slotName>`
+  media: 'fui-TeachingPopoverBody__media',
 };
 
-/**
- * Styles for the root slot
- */
+export const useMediaStyles = makeStyles({
+  base: {
+    ...shorthands.gridArea('media'),
+    ...shorthands.overflow('hidden'),
+    width: '288px',
+    marginBottom: '12px',
+    verticalAlign: 'middle',
+    justifyContent: 'center',
+    display: 'flex',
+    '& > img': {
+      objectFit: 'cover',
+      display: 'inline-block',
+    },
```

Comment:
Let's use `<Image />` component in examples that supports there features instead of custom CSS